### ISSUE TITLE
Issue building modules

### ISSUE THREAD

**Issue by wyardley** [terraform-google-modules/terraform-google-kubernetes-engine#2097]
### TL;DR

I started getting the following error on OS X when running `make build` or `make `docker_generate_modules`

Since it's all in Docker (I think from https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit?), debugging this or getting any more information on what's going wrong is a little challenging.

**Edit:** I eventually made the problem go away by [removing the repo and re-cloning](https://xkcd.com/1597/). However, `git status` hadn't shown any unexpected files present 🤷 

### Observed behavior

```
docker run --rm -it \
                -v "/Users/wby/git/terraform-google-kubernetes-engine":/workspace \
                gcr.io/cloud-foundation-cicd/cft/developer-tools:1.22 \
                /bin/bash -c 'source /usr/local/bin/task_helper_functions.sh && generate_modules'
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Traceback (most recent call last):
  File "/usr/local/bin/generate_modules.py", line 80, in <module>
    main(sys.argv)
  File "/usr/local/bin/generate_modules.py", line 53, in main
    template = env.get_template(template_file)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/jinja2/environment.py", line 1013, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/jinja2/environment.py", line 972, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/jinja2/loaders.py", line 210, in get_source
    contents = f.read()
               ^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xda in position 17: invalid continuation byte
Running terraform fmt
```

### Additional information
The platform warning has been ongoing for a bit - not sure if a PR to add an explicit `--platform=linux/amd64` would break things for users on any other platforms.

**Comment by wyardley** [comment]
@apeabody just FYI. I'm closing this since I did manage to resolve it.
